projects/element-ng/icon/si-icon-next.component.ts

```diff
@@ -91,7 +91,7 @@ export class SiIconNextComponent {
    *
    * @example
    * ```ts
-   * import { elementUser } from '@simpl/element-icons/ionic';
+   * import { addIcons, elementUser } from '@siemens/element-ng/icon'
```

Comment:
those imports are fine here. I was wrong the first time. One should not use the siemens icon thing here.

We actuall have to find a long term solution.